.github/workflows/build-test.yaml

```diff
@@ -431,10 +431,12 @@ jobs:
         with:
           test-focus: 'Regression'
           k8s-version: 'v1.26.1-k3s1'
+          platform: 'k3s'
           testim-branch: ${{ github.head_ref == 'main' && 'master' || github.head_ref }}
           testim-access-token: '${{ secrets.TESTIM_ACCESS_TOKEN }}'
           aws-access-key-id: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_ACCESS_KEY_ID }}'
           aws-secret-access-key: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_SECRET_ACCESS_KEY }}'
+          gke-service-key: ${{ secrets.GOOGLE_CLOUD_CREDENTIALS }}
```

Comment:
feels weird that this is passed to every test even if we don't use it for most.  though I can see how it makes it easy to simply toggle or add new platforms.  if you don't pass this, will it just fail since it's not a required input?

Reply:
Yeah, if the credentials aren't passed, the test will fail with an error about being unable to authenticate to the google cloud, which should be obvious to anyone adding gke tests in the future. I've removed all the unneeded `gke-service-key` entries as it does feel weird passing an unused secret